spec/controllers/faculty_access_controller_spec.rb

```diff
@@ -1,33 +0,0 @@
-require 'rails_helper'
-
-RSpec.describe FacultyAccessController, type: :controller do
-  let(:user) do
-    create_user('user')
-  end
-
-  before do
-    controller.sign_in!(user)
-  end
-
-  context 'when the educator feature flag is OFF' do
-    before do
-      Settings::FeatureFlags.educator_feature_flag = false
-    end
-
-    it 'renders apply form' do
-      get(:apply)
-      expect(response).to render_template(:apply)
-    end
-  end
-
-  context 'when the educator feature flag is ON' do
-    before do
-      Settings::FeatureFlags.educator_feature_flag = true
-    end
-
-    it 'redirects to the SheerID form' do
-      get(:apply)
-      expect(response).to redirect_to(educator_sheerid_form_path)
```

Comment:
This behavior still exists on production, btw.  Keep test?

Reply:
I think we need to remove these [routes](https://github.com/openstax/accounts/blob/main/config/routes.rb#L172-L175) or just add a match/redirect in routes. Honestly, I can't imagine why anyone/thing would be hitting the /apply route after two years of this being released.